roles/node_exporter/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: 'set node exporter repository'
+  yum_repository:
+    name: prometheus-rpm_release
+    baseurl: https://packagecloud.io/prometheus-rpm/release/el/$releasever/$basearch
+    description: prometheus-rpm_release
+    enabled: True
+    gpgcheck: False
+    gpgkey: https://packagecloud.io/prometheus-rpm/release/gpgkey
+    sslverify: True
+    sslcacert: /etc/pki/tls/certs/ca-bundle.crt
+    repo_gpgcheck: True
+    state: present
+
+# https://github.com/ansible/ansible/issues/20711
```

Comment:
@ballot-scality I'm not sure how to detect that the key has been accepted or not.